src/Planner/CollectSets.cpp

```diff
@@ -24,6 +24,34 @@ namespace ErrorCodes
     extern const int UNSUPPORTED_METHOD;
 }
 
+QueryTreeNodePtr makeExecutableSubqueryForIn(const QueryTreeNodePtr & in_second_argument, const ContextPtr & context)
```

Comment:
This method needs to be at the end of this file, because it is last in the interface file.